scripts/complete_liquidity_provision.js

```diff
@@ -105,15 +116,29 @@ module.exports = async (callback) => {
       callback("Error: Choose a smaller fleetSize, otherwise your payload will be to big for Infura nodes")
     }
 
-    console.log(`2. Deploying ${argv.fleetSize} trading brackets`)
-    const bracketAddresses = await deployFleetOfSafes(masterSafe.address, argv.fleetSize, true)
-    console.log("List of bracket traders in one line:", bracketAddresses.join())
+    let bracketAddresses
+    if (argv.brackets.length > 0) {
+      console.log("==> Skipping safe deployment and using brackets safeOwners")
+      bracketAddresses = argv.brackets
+      // Ensure that safes are all owned solely by masterSafe
+      for (const safeAddr of argv.brackets) {
+        const safeOwners = await (await GnosisSafe.at(safeAddr)).getOwners()
+        // TODO: write function like safeOwnedSolelyBy(safe, owner) -> Bool
+        if (!(safeOwners.length === 1 && safeOwners.includes(masterSafe.address))) {
+          callback(`Error: Bracket ${safeAddr} is not owned (or at least not solely) by master safe ${masterSafe.address}`)
+        }
+      }
```

Comment:
Should we extract all these check into one method? I think Federico wanted to verify as well that there are not yet any orders.

Reply:
@fedgiac - would you agree like to see that the following snippet of code be made into its own function?

```js
      // Ensure that safes are all owned solely by masterSafe
      await Promise.all(
        bracketAddresses.map(async (safeAddr) => {
          if (!(await isOnlySafeOwner(masterSafe.address, safeAddr))) {
            callback(`Error: Bracket ${safeAddr} is not owned (or at least not solely) by master safe ${masterSafe.address}`)
          }
        })
      )
```